Update README.adoc (#24)
EMnify/Documentation: README.adoc

=== Identity and Access Management

=== Embedded Analytics
Currently in our link:http://cdn.emnify.net/[EUI Portal] and our link:https://portal.emnify.com/[new Portal] we provide Embedded Analytics - powered by a tool called Periscope. Depending on the customer type, the analytics provides the customer with dashboards such as: 

*CDR Export
*Data Traffic Per Ratezone
*Data Usage by Service Profile
*Endpoint Repository
*Global Distribution
*KPI Number of SIMs online
*Live Traffic
*Network Activity
*Network Utilisation
*Overview Current Month
*SIM Database
*Traffic History
*Traffic by Mobile Network
*Usage by Endpoint

> With the release of the new portal, ourt teams are also in the process of evaluating new tools to completely redesign the embedded dashboards from the ground up which we hope will provide our customers with more value and targeted insights for their businesses.

Reviewer: ```suggestion
We are constantly evaluating new tools to enhance the embedded dashboards and hope to provide our customers with more value and targeted insights for their businesses.
```
